deb/brave-keyring/DEBIAN/postinst

```diff
@@ -1,4 +1,14 @@
-#!/bin/sh
+#!/bin/sh -e
 
-# Delete (expired) key added by old install instructions
-apt-key --keyring /etc/apt/trusted.gpg del D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821
```

Comment:
We manage the removal of old keys here. If we plan to stop doing that we should run it by @fmarier

Reply:
I was the one who suggested that. My thinking is that it's been more than a year since we updated our instructions to use the separate keyring in `/etc/apt/trusted.gpg.d/brave-browser-release.gpg`. The new (well, not so new anymore) instructions don't need this since we simply overwrite the file.

Reply:
A year seems like more than enough.